pkg/apb/client.go

```diff
@@ -200,8 +194,5 @@ func (c *Client) refreshLoginToken(clusterConfig ClusterConfig) error {
 	if err != nil {
 		return err
 	}
-
-	c.log.Debug("No error reported after running oc login. Cmd output:")
-	c.log.Debug(string(output))
```

Comment:
But this is really really useful, especially for debugging. I guess we could add it when we're debugging.